app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -417,6 +426,7 @@ private boolean useResizingImageView() {
     }
 
     private void initView(Context context) {
+        setId(AndroidUtil.generateViewId());
```

Comment:
why is this required ?

Reply:
If not set, `this.getId()` call above returns -1.

Reply:
Ahh Makes sense.